src/R/Editor/Impl/Selection/SelectWordCommand.cs

```diff
@@ -24,60 +22,13 @@ public sealed class SelectWordCommand : ViewCommand {
         public override CommandResult Invoke(Guid group, int id, object inputArg, ref object outputArg) {
```

Comment:
We still call it on double click since that thing does not call structure navigator...